tests/testJointcal.py

```diff
@@ -0,0 +1,239 @@
+        self.src = src
+        self.calexp_md = calexp_md
+        # info about our "telescope"
+        # self.calexp_md.setDouble("AIRMASS", 1)
+        self.calexp_md.setDouble("MJD", 54321.)
+        # self.calexp_md.setDouble("EXPTIME", 30.)
+        self.calexp_md.setDouble("LST", 53.00914)
+        self.calexp_md.setDouble("RA2000", 53.00914)
+        self.calexp_md.setDouble("DEC2000", -27.43895)
+
+        self.dataId = dataId
+        self.dataId['ccd'] = 1
+        self.dataId['tract'] = 1
+
+    def get(self, name, immediate=True):
+        return getattr(self, name)
+
+    def put(self, value, name):
+        setattr(self, name, value)
+
+    def getButler(self):
+        class NamedThing(object):
+            def __init__(self, name):
+                self.name = name
+
+            def getName(self):
+                return self.name
+
+        return {'camera': NamedThing('monkeySim')}
+
+
+class JointcalTest(lsst.utils.tests.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        """Can only do the eups setup once, so have to do it here only."""
+
+        # NOTE: this may be cheating? How else should I do this?
+        jointcal_reference_dir = os.path.join(os.environ['VALIDATION_DATA_JOINTCAL_DIR'],
+                                              'twinkles1_and_index')
+        eups.setup('astrometry_net_data', productRoot=jointcal_reference_dir)
+
+    def setUp(self):
+
+        data_dir = os.environ['VALIDATION_DATA_JOINTCAL_DIR']
+
+        # position of the Twinkles run 1 catalog
+        center = coord.IcrsCoord(53.00914*geom.degrees, -27.43895*geom.degrees)
+        r = geom.Angle(3, geom.degrees)
+        config = astrom.LoadAstrometryNetObjectsTask.ConfigClass()
+        refLoader = astrom.LoadAstrometryNetObjectsTask(config=config)
+        reference = refLoader.loadSkyCircle(center, r, filterName='r')
+        self.reference = reference.refCat.copy()  # for in-memory contiguity.
+
+        # Get a set of source catalogs.
+        self.catalogs = []
+        butler = persistence.Butler(os.path.join(data_dir, 'twinkles1'))
+        for visit in butler.queryMetadata('src', 'visit'):
+            src = butler.get('src', dataId={'visit': visit})
+            dataId = butler.dataRef('src', visit=visit).dataId
+            calexp_md = butler.get('calexp_md', {'visit': visit, 'raft': '2,2', 'sensor': '1,1'})
+            self.catalogs.append(FakeRef(src, calexp_md, dataId))
+
+        self.jointcalTask = jointcal.JointcalTask()
+
+    @unittest.skip('jointcal currently fails if only given one catalog!')
+    def testJointCalTask_1_catalog(self):
+        self.jointcalTask.run(self.catalogs[:1])
+
+    def testJointCalTask_2_catalog(self):
+        ncat = 2
+        self.jointcalTask.run(self.catalogs[:ncat])
+        rms_rel, rms_abs = compute_rms(self.reference, self.catalogs[:ncat], fluxlimit=100)
+        # NOTE: this number come from an initial jointcal run.
+        self.assertLess(rms_rel, 8.4e-3/206265)
```

Comment:
Better yet, use a named constant. Since you divide by this value a lot! Then you can skip the comment.